build-uclibc.sh

```diff
@@ -339,13 +344,19 @@ cd "${build_dir_stage1}"
 # Configure the build. Disable anything that might try to build a run-time
 # library and don't bother with multilib for stage 1. Note: with gcc 4.4.x, we
 # also disable building libgomp
+if [ "x${TLS_SUPPORT}" = "xyes" ]
+then
+    thread_flags=""
```

Comment:
I think setting this to `--enable-threads --enable-tls` would be better, to avoid dependencies on default behaviour of configure.